courses/fundamentals_of_ada/010_overview.rst

```diff
@@ -707,3 +708,25 @@ Canonical First Program
 
    - :command:`say_hello` (Windows)
    - :command:`./say_hello` (Linux/Unix)
+
+--------------------------------
+"Hello World" Lab - GNATstudio
```

Comment:
Aren't labs supposed to go into their own file ?

Reply:
This is more a "Setup Check" than an actual lab, which is why I embed it in the lecture. Generally, this is done together to make sure everyone is at the same point before going on